static/app/views/replays/replayTable.tsx

```diff
@@ -0,0 +1,182 @@
+        'project',
+        'max(timestamp)',
+        'min(timestamp)',
+        'equation|max(timestamp)-min(timestamp)',
+        'count_if(event.type,equals,error)',
+      ],
+      orderby: '-min_timestamp',
+      environment: selection.environments,
+      projects: selection.projects,
+      query: `(title:sentry-replay-event OR event.type:error) AND (${query})`,
+    };
+
+    if (selection.datetime.period) {
+      eventQueryParams.range = selection.datetime.period;
+    }
+    return EventView.fromNewQueryWithLocation(eventQueryParams, location);
+  };
+
+  return (
+    <DiscoverQuery
+      eventView={getEventView()}
+      location={location}
+      orgSlug={organization.slug}
+    >
+      {data => {
+        const dataEntries = data.tableData
+          ? mergeReplayEntries(data.tableData?.data as Replay[], 'replayId')
+          : null;
+        return replayList?.map(replay => {
+          return (
+            <Fragment key={replay.id}>
+              <ReplayUserBadge
+                avatarSize={32}
+                displayName={
+                  <Link
+                    to={`/organizations/${organization.slug}/replays/${generateEventSlug({
+                      project: replay.project,
+                      id: replay.id,
+                    })}/`}
+                  >
+                    {replay['user.display']}
+                  </Link>
+                }
+                user={{
+                  username: replay['user.display'],
+                  id: replay['user.display'],
+                  ip_address: replay['user.display'],
+                  name: replay['user.display'],
+                  email: replay['user.display'],
+                }}
+                // this is the subheading for the avatar, so displayEmail in this case is a misnomer
+                displayEmail={getUrlPathname(replay.url) ?? ''}
+              />
+              {isScreenLarge && (
+                <StyledPanelItem>
+                  <ProjectBadge
+                    project={
+                      projects.find(p => p.slug === replay.project) || {
+                        slug: replay.project,
+                      }
+                    }
+                    avatarSize={16}
+                  />
+                </StyledPanelItem>
+              )}
+              <StyledPanelItem>
+                <TimeSinceWrapper>
+                  {isScreenLarge && (
+                    <StyledIconCalendarWrapper color="gray500" size="sm" />
+                  )}
+                  <TimeSince date={replay.timestamp} />
+                </TimeSinceWrapper>
+              </StyledPanelItem>
+              {data.tableData && dataEntries ? (
+                <React.Fragment>
+                  <React.Fragment>
+                    <StyledPanelItem>
+                      <Duration
+                        seconds={
+                          dataEntries[replay.id]
+                            ? dataEntries[replay.id]['equation[0]'] / 1000
```

Comment:
I don't think there's a need to divide by 1000 here, the response is in seconds.